docs/reference/search/search.asciidoc

```diff
@@ -146,8 +154,17 @@ level settings.
 include::{docdir}/rest-api/common-parms.asciidoc[tag=search_type]
 
 `seq_no_primary_term`::
-  (Optional, boolean) If `true`, returns sequence number and primary term of the
-  last modification of each hit.
++
+--
+(Optional, boolean) If `true`, returns sequence number and primary term of the
+last modification of each hit. See <<optimistic-concurrency-control>>.
+
+[IMPORTANT]
+====
```

Comment:
I was just getting up to speed on the search docs improvements, they look great so far! One small comment: I wonder if we could pull these 'tips' into an overall note that says something like 'several options can be provided both as query parameters and in the request body. If the option is specified in both places, only the query parameter is used.'

Reply:
++ Thanks for commenting @jtibshirani.

These notes are adding some unneeded bloat. I like the idea of adding a general note to the parameters section.

We can also add a footnote at the param level that links out without getting in the way too much. That way if users skip the note while searching for a particular param, they'll still get an indicator.

<img width="770" alt="Screen Shot 2020-06-01 at 5 04 52 PM" src="https://user-images.githubusercontent.com/40268737/83454430-2dab3c80-a42a-11ea-85ae-099694279cff.png">

Reply:
Related PR: #57491

Reply:
Would there be a reference + footnote for each parameter section? For me even this might be too much -- it seems very unusual (and in fact would be a bad practice) to pass the same parameter in two places. I'm not even sure why we have so much overlap in the API between query and request parameters. Maybe it's best to not emphasize it and just think of the 'IMPORTANT: ...' note you added as helpful debugging information when you accidentally specify the parameter in two places.

Reply:
Thanks for the feedback @jtibshirani. My only concern would be people accidentally skipping the `IMPORTANT` note, but you're right --- ideally users shouldn't be specifying both in any case. Also agree that removing some of the overlap would be best in the long term.

I'll revise #57491 to only use an important note and ditch the footnote plan. Thanks again for your feedback!